packages/exp/src/commands/generate-unimodule.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @flow
+ */
+
+export default (program: any) => {
+  program
+    .command('generate-unimodule')
+    .alias('gum')
+    .description('Generate new unimodule.')
+    .asyncAction(async () => {
+      const readline = require('readline');
+      const proc = require('child_process');
+      const path = require('path');
```

Comment:
Imports should be in the beginning of the file and use the ES2015 syntax like this:
```js
import readline from 'readline';
import childProcess from 'child_process';
import path from 'path';
```